src/frontend/web_application/src/components/form/Switch/index.jsx

```diff
@@ -2,26 +2,30 @@ import React, { PropTypes } from 'react';
 import { v1 as uuidV1 } from 'uuid';
 import './style.scss';
 
-const Switch = ({ label, ...inputProps }) => {
+const Switch = ({ label, duplicateLabel, ...inputProps }) => {
   const id = uuidV1();
 
   return (
-    <div className="m-switch">
-      <input
-        type="checkbox"
-        className="m-switch__input"
-        id={id}
-        {...inputProps}
-      />
-      <label className="m-switch__paddle" htmlFor={id}>
-        <span className="show-for-sr">{label}</span>
-      </label>
+    <div>
+      <div className="m-switch">
+        <input
+          type="checkbox"
+          className="m-switch__input"
+          id={id}
+          {...inputProps}
+        />
+        <label className="m-switch__paddle" htmlFor={id}>
+          <span className="show-for-sr">{label}</span>
+        </label>
+      </div>
+      { duplicateLabel && <label htmlFor={id} className="m-switch__label">{label}</label> }
```

Comment:
I would prefer a `SwitchFieldGroup` component instead. Here we have an Element outside of the Block